xarray/tests/test_plot.py

```diff
@@ -267,6 +268,35 @@ def test2d_1d_2d_coordinates_contourf(self):
         a.plot.contourf(x="time", y="depth")
         a.plot.contourf(x="depth", y="time")
 
+    def test_contourf_cmap_set(self):
+        sz = (4, 4)
+        a = DataArray(easy_array(sz), dims=["z", "time"])
```

Comment:
```suggestion
        a = DataArray(easy_array(shape), dims=["z", "time"])
```

Reply:
I'd copied the `sz = (4,4)` from the test just above this one... Actually the variable is not really necessary as it's only used once. Probably better to just change here to
```suggestion
        a = DataArray(easy_array((4, 4)), dims=["z", "time"])
```